SEIR/seir.py

```diff
@@ -20,13 +35,21 @@ def onerun_SEIR(uid, s):
 
     npi = NPI.NPIBase.execute(npi_config=s.npi_config, global_config=config, geoids=s.spatset.nodenames)
     npi = npi.get().T
+    print(s.npi_config)
+    r_assign('ti_str', str(s.ti))
+    r_assign('tf_str', str(s.tf))
+    r_assign('foldername', '../data/ch/')
+    r_assign('setupname', s.setup_name)
+    r_source('NPI/NPI_for_baseline.R')
+    R0 = robjects.r['NPI'].T
 
     seeding = setup.seeding_draw(s, uid)
 
     mobility_geoid_indices = s.mobility.indices
     mobility_data_indices = s.mobility.indptr
     mobility_data = s.mobility.data
-    parameters = setup.parameters_quick_draw(config["seir"]["parameters"], len(s.t_inter), s.nnodes, s.dt, npi)
+    parameters = setup.parameters_quick_draw(config["seir"]["parameters"],
+                     config, len(s.t_inter), s.nnodes, s.dt, R0, s.setup_name)
```

Comment:
I think justin wants us to move away from the model of passing the config everywhere.  For the NPIs it's necessary, but here it's not so much i think